### ISSUE TITLE
Multi-task learning: Different Hyper-params for different task

### ISSUE THREAD

**Issue by HenryYuen128** [UKPLab/sentence-transformers#1413]
Is it possible to assign different groups of hyper-params(e.g. learning rate, epoch) for every task in multi-task learning?

**Comment by nreimers** [comment]
This is sadly not possible
